carddav/server.go

```diff
@@ -497,6 +497,14 @@ func (b *backend) propFindAddressBook(ctx context.Context, propfind *internal.Pr
 				{ContentType: vcard.MIMEType, Version: "4.0"},
 			},
 		}),
+		internal.CurrentUserPrivilegeSet: func(raw *internal.RawXMLValue) (interface{}, error) {
```

Comment:
`internal.PropFindValue` can be used here.

The same should be applied to caldav.